container/push.bzl

```diff
@@ -131,9 +131,9 @@ def _impl(ctx):
         ),
```

Comment:
Does this now not also need to also return an
`OutputGroupInfo(digest = depset([digest_output])`
as indicated in https://github.com/bazelbuild/bazel/issues/7977